src/components/ArtistProtectedIndex.js

```diff
@@ -0,0 +1,41 @@
+import React from "react"
+import { NavLink } from "react-router-dom" 
+
+const ArtistProtectedIndex = ({ currentUser, apartments }) => {
```

Comment:
This looks like code copied from apartment app that is not going to match your data structure based on the mock data you provided. Also I lean towards this being in the page directory instead of the component directory.